examples/flutter_gallery/lib/gallery/item.dart

```diff
@@ -67,6 +67,13 @@ final List<GalleryItem> kAllGalleryItems = <GalleryItem>[
     routeName: ContactsDemo.routeName,
     buildRoute: (BuildContext context) => new ContactsDemo(),
   ),
+  new GalleryItem(
+    title: 'Animation',
```

Comment:
you'd have to change test_driver/transitions_perf_test.dart too. Just happened to me :)